queue_job/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Job Queue",
-    "version": "14.0.1.3.2",
+    "version": "14.0.1.3.3",
```

Comment:
this is more than a patch

Reply:
good day @simahawk , so bumping the minor would be better `14.0.1.4.0`

Reply:
yes, thanks